src/frequenz/sdk/config/_manager.py

```diff
@@ -29,6 +30,7 @@ def __init__(  # pylint: disable=too-many-arguments
         force_polling: bool = True,
         name: str | None = None,
         polling_interval: timedelta = timedelta(seconds=5),
+        wait_for_first_timeout: timedelta = timedelta(seconds=5),
     ) -> None:
         """Initialize this config manager.
```

Comment:
Yeah, like a library to interact with it more easily. I wouldn't say is a wrapper, is not really wrapping it. I would say more like a client, but we don't call them clients, neither in the `frequenz.sdk.microgrid` package nor in `dispatch` for example (`Dispatcher` is also a client/wrapper for the [`DispatchingActor`](https://github.com/frequenz-floss/frequenz-dispatch-python/blob/v0.x.x/src/frequenz/dispatch/actor.py#L25C7-L25C23). Users normally don't even need to know there is an actor under the hood, that's just implementation detail. With config the problem is we didn't have a proper library so we started to use the actor and now we can't "unknow" the actor is there, so it might add to the confusion :D